autoload/vundle/installer.vim

```diff
@@ -6,9 +6,13 @@
 " ...    -- any number of bundle specifications (separate arguments)
 " ---------------------------------------------------------------------------
 func! vundle#installer#new(bang, ...) abort
-  let bundles = (a:1 == '') ?
-        \ g:bundles :
-        \ map(copy(a:000), 'vundle#config#bundle(v:val, {})')
+  if a:0 == 0
+    let bundles = g:bundles
+  else
```

Comment:
Can you please break this into two branches with`elseif (a:bang)` and add come comments as to which cases each branch covers.